pkg/webhookconfig/configmanager.go

```diff
@@ -388,39 +372,30 @@ func (m *webhookConfigManager) reconcileWebhook(namespace, name string) error {
 	return nil
 }
 
-func (m *webhookConfigManager) getPolicy(namespace, name string) (*kyverno.ClusterPolicy, error) {
+func (m *webhookConfigManager) getPolicy(namespace, name string) (policy, error) {
 	if namespace == "" {
 		return m.pLister.Get(name)
+	} else {
+		return m.npLister.Policies(namespace).Get(name)
 	}
-
-	nsPolicy, err := m.npLister.Policies(namespace).Get(name)
-	if err == nil && nsPolicy != nil {
-		p := kyverno.ClusterPolicy(*nsPolicy)
-		return &p, err
-	}
-
-	return nil, err
 }
 
-func (m *webhookConfigManager) listAllPolicies() ([]*kyverno.ClusterPolicy, error) {
-	policies := []*kyverno.ClusterPolicy{}
-
+func (m *webhookConfigManager) listAllPolicies() ([]policy, error) {
+	policies := []policy{}
 	polList, err := m.npLister.Policies(metav1.NamespaceAll).List(labels.Everything())
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to list Policy")
 	}
-
-	for _, pol := range polList {
-		p := kyverno.ClusterPolicy(*pol)
-		policies = append(policies, &p)
+	for _, p := range polList {
```

Comment:
Not sure why `policies = append(policies, polList...)` does not compile :thinking: